autogpt_platform/backend/backend/copilot/dream/fetch.py

```
@@ -51,6 +51,55 @@ class FactRow(BaseModel):
+    last_recalled_at: str | None = None  # ISO timestamp
+    prev_recalled_at: str | None = None  # ISO timestamp
+
+    @classmethod
+    def usage_only(
+        cls,
+        *,
+        uuid: str,
+        recall_count: int | None,
+        last_recalled_at: str | None,
+        prev_recalled_at: str | None,
+    ) -> "FactRow":
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)